frontend/src/use-cases/home/home.js

```diff
@@ -1,23 +1,19 @@
-import React, { useState, useEffect } from "react";
+import React, { useContext } from "react";
 
-import "./home.css";
-import Prompt from "./views/Prompt/prompt.view";
+import Prompt from "./views/Prompt";
 import SuggestionBoard from "./views/SuggestionBoard/SuggestionBoard";
-import { checkLogin } from "../../services/data.service";
+import UserContext from "common/context/user-context";
+import { HomeWrapper, Divider } from "./home.style";
 
 const Home = () => {
-    const [suggestionboard, setSuggestionBoard] = useState(<div />);
-
-    useEffect(() => {
-        checkLogin().then(() => setSuggestionBoard(<SuggestionBoard />));
-        return () => {};
-    }, []);
+    const [user] = useContext(UserContext);
 
     return (
-        <div className="main">
+        <HomeWrapper>
             <Prompt />
-            {suggestionboard}
-        </div>
+            <Divider />
```

Comment:
rdc has a `DigitDesign.Divider`

Reply:
![image](https://user-images.githubusercontent.com/31804935/81729986-2b8a3980-948d-11ea-9548-e8b31ffa60af.png)
The divider is broken..